src/update_flash.c

```diff
@@ -492,6 +504,25 @@ static int RAMFUNCTION wolfBoot_update(int fallback_allowed)
                 wolfBoot_set_update_sector_flag(sector, flag);
         }
         sector++;
+        /* headers that can be in different positions depending on when the
```

Comment:
Should this be wrapped in `#ifndef DISABLE_BACKUP`?

Reply:
it already is, that entire section of code is wrapped with that definition